src/pages/EnablePayments/AddBankAccount/substeps/Confirmation.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import DotIndicatorMessage from '@components/DotIndicatorMessage';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PersonalBankAccountForm} from '@src/types/form';
+import INPUT_IDS from '@src/types/form/PersonalBankAccountForm';
+import type {PlaidData, UserWallet} from '@src/types/onyx';
+
+type ConfirmationOnyxProps = {
+    /** The draft values of the bank account being setup */
+    personalBankAccountDraft: OnyxEntry<PersonalBankAccountForm>;
+
+    /** Contains plaid data */
+    plaidData: OnyxEntry<PlaidData>;
+
+    /** The user's wallet */
+    userWallet: OnyxEntry<UserWallet>;
+};
+
+type ConfirmationProps = ConfirmationOnyxProps & SubStepProps;
+
+const BANK_INFO_STEP_KEYS = INPUT_IDS.BANK_INFO_STEP;
+const BANK_INFO_STEP_INDEXES = CONST.WALLET.SUBSTEP_INDEXES.BANK_ACCOUNT;
+
+function Confirmation({personalBankAccountDraft, plaidData, onNext, onMove, userWallet}: ConfirmationProps) {
```

Comment:
ConfirmationStep would be better name here, I think we could add the `Step` suffix to all these components